src/overrides.ts

```diff
@@ -0,0 +1,14 @@
+import * as vscode from 'vscode';
+
+const vscodeShowErrorMessage = vscode.window.showErrorMessage;
+
+async function calvaShowErrorMessage(message: string) {
+  const exlusionRegexps = [/Request textDocument\/codeAction failed./];
+  if (!exlusionRegexps.some((re) => re.test(message))) {
+    return vscodeShowErrorMessage(message);
+  }
+}
+
+export function activate() {
```

Comment:
This should return a disposable, that can be pushed into `context.subcriptions`, to restore built in `showErrorMessage`.

Reply:
Are you suggesting we create a class that implements `Disposable`, and then create a dispose method that restores the built in `showErrorMessage`?

Considering I don't think this affects any other extensions, I don't see the need for that, but if we did want to restore the built-in `showErrorMessage`, I'd rather add a `deactivate` function. So in essence, we're activating and deactivating overrides.

Reply:
> This not about potentially making this change optional for users.

I wasn't suggesting optionality for users, just to be clear.

>  It about being good citizens, when for example, someone uninstalls the extension.

If I thought this would affect other extensions, I wouldn't make this change, at least not as it is, and maybe not at all.

> All of the numerous calls to context.subcriptions.push in extension.ts are to provide disposables to VS Code.

I'm aware of this. I've been a maintainer of Calva for quite some time now 😃. I appreciate your reviewing and concerns, though.

Also, thanks for reminding me that we don't need a class and can just use an object literal with a `dispose` property.